app/models/user.rb

```diff
@@ -1,10 +1,17 @@
 class User < ApplicationRecord
-  has_many :posts, foreign_key: :author_id
-  has_many :comments
-  has_many :likes
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
Good job for uncommenting the required attributes However there's one attribute that is still commented which is `:confirmable` and it's part of the project requirement. Kindly uncomment that and use it with devise so that the user can confirm the email. 👍 

Also, Kindly remove the redundant attributes. It seems like you have repeated data in your `user.rb` file.

<img width="399" alt="Screenshot 2023-04-15 at 6 41 59 AM" src="https://user-images.githubusercontent.com/83048208/232177134-6a27c0e9-d610-4cab-bf3a-77026da55e7a.png">